src/lib/analysers/ProtestsPretrained/core.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+import json
+import os
+import torch
+from torch.autograd import Variable
+from PIL import Image
+from utils import transform, modified_resnet50, decode
+
+from importlib import import_module
+from lib.common.exceptions import InvalidAnalyserConfigError
```

Comment:
Can delete these two lines, as they're not being used anywhere.

Reply:
Removed!